### PR TITLE
RTL, Arabic, Hebrew support

### PR DESCRIPTION
**Pull request by aardrian**
The arrow sits on the right, but in a native select in Arabic, Hebrew, or any right-to-left text the arrow is on the left in a native unstyled select.

### CODE REVIEW

**Comment by scottjehl** [comment]
We love this PR, thanks! My memory is a little foggy on common RTL patterns, but is there a selector or approach that is fairly comprehensive for RTL languages or is it typical to write selectors to each relevant language? I'm curious why we'd explicitly mention a couple languages but not all, unless there's not a simple pattern for doing that. Also, is the `.select-css:dir(rtl)` attr worth adding for future support?

**Comment by aardrian** [comment]
Frankly, I have not been immersed in internationalization / localization work in almost a decade. So this is me testing with modern browsers while using old experience and limited knowledge of current best practices.

Experience says developers may have a section of the page in a different language, but may not set the `dir` attribute, or may translate an entire page without the `dir` attribute. So I have selectors in there for two major RTL languages.

Some developers may do neither and just let the browser default to the user's locale / language settings. These may be common in places where the audience and developer all work in the same language.

`select-css:dir(rtl)` may be worth adding, but that would also require more testing. My guess is if you add that, IE11 will ignore the entire selector, so that may have to live as its own selector with redundant styles.

In other words, I suggest you find an internationalization expert who has experience in RTL languages. In the meantime, my code will at least address the issue today while you are finding that expert.

**Comment by ahmadalfy** [comment]
I've been working with internationalization for more than a decade. We definitely use the direction attribute on the `html` tag and we target elements using the `dir` attribute. There is a new pseudo-class `:dir(rtl)` but it is not widely supported yet (only on [Firefox](https://caniuse.com/#feat=css-dir-pseudo)). Thank you @aardrian for the PR, this is fair and would be definitely helpful for anyone who writes RTL layouts.

**Comment by StommePoes** [comment]
Is "iw" more supported than "he"??

**Comment by aardrian** [comment]
> Is "iw" more supported than "he"??

Supported by what?

**Comment by StommePoes** [comment]
Browsers. I didn't recognise "iw" so I looked it up.
